topostats/run_topostats.py

```diff
@@ -163,12 +164,12 @@ def main(args=None):
 
     # Write sample configuration if asked to do so and exit
     if args.create_config_file:
-        write_yaml(
-            config,
-            output_dir="./",
-            config_file=args.create_config_file,
-            header_message="Sample configuration file auto-generated",
-        )
+        default_config_path = pkg_resources.path(__package__, "default_config.yaml")
+        if ".yaml" not in args.create_config_file and ".yml" not in args.create_config_file:
+            create_config_path = "./" + args.create_config_file + ".yaml"
```

Comment:
Not tested this on M$-Win but would the hard coding of `./` cause any problems? I've a vague memory that the Windows Linux Shell does interpret/handle these correctly. If not perhaps using `pathlib.Path("./")` might mitigate the problem (if there is one!).